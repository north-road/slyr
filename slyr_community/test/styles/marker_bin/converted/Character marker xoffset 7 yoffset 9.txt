[{'class': 'QgsSimpleMarkerSymbolLayer',
  'color': '0,0,0,255',
  'joinstyle': 'miter',
  'name': 'square',
  'offset': '7,-9',
  'offset_unit': 'Point',
  'outline_style': 'no',
  'size': '5.89088',
  'size_unit': 'Point'}]
