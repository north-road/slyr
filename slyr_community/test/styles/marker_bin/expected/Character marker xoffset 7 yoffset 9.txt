{'halo': False,
 'halo_size': 2.0,
 'halo_symbol': {'color': {'B': 255,
                           'G': 255,
                           'R': 255,
                           'dither': True,
                           'is_null': False,
                           'stream_offset': '0xdc',
                           'type': 'RgbColor',
                           'version': 1},
                 'enabled': True,
                 'fill_style': 'solid',
                 'locked': False,
                 'outline': {'color': {'B': 110,
                                       'G': 110,
                                       'R': 110,
                                       'dither': True,
                                       'is_null': False,
                                       'stream_offset': '0x99',
                                       'type': 'RgbColor',
                                       'version': 1},
                             'enabled': True,
                             'line_type': 'solid',
                             'locked': False,
                             'stream_offset': '0x87',
                             'symbol_level': 0,
                             'type': 'SimpleLineSymbol',
                             'version': 1,
                             'width': 0.0},
                 'stream_offset': '0x75',
                 'symbol_level': 0,
                 'type': 'SimpleFillSymbol',
                 'version': 1},
 'levels': [{'angle': 0.0,
             'color': {'B': 0,
                       'G': 0,
                       'R': 0,
                       'dither': False,
                       'is_null': False,
                       'stream_offset': '0x12d',
                       'type': 'RgbColor',
                       'version': 1},
             'enabled': True,
             'font': 'ESRI Default Marker',
             'locked': False,
             'rotate_with_transform': True,
             'size': 8.0,
             'std_font': {'charset': 0,
                          'font_name': 'ESRI Default Marker',
                          'italic': False,
                          'size': 8.25,
                          'stream_offset': '0x1e2',
                          'strikethrough': False,
                          'type': 'Font',
                          'underline': False,
                          'version': 1,
                          'weight': 400},
             'stream_offset': '0x11b',
             'symbol_level': 0,
             'type': 'CharacterMarkerSymbol',
             'unicode': 34,
             'version': 4,
             'x_offset': 7.0,
             'x_scale': 1.0,
             'y_offset': 9.0,
             'y_scale': 1.0}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerMarkerSymbol',
 'version': 3}
